examples/models/tile_source.py

```diff
@@ -1,3 +1,11 @@
+''' Example to demonstrate creating map-based visualizations and 
+working with geographical data using WMTSTileSource in Bokeh. 
+
+.. bokeh-example-metadata::
+    :apis: bokeh.models.WMTSTileSource, bokeh.models.Range1d, bokeh.models.BoxZoomTool, bokeh.models.PanTool, bokeh.models.WheelZoomTool
+    :refs: :ref:`ug_topics_tile_provider_maps`
+    :keywords: tile source
+'''
```

Comment:
Formatting consistency with other examples (blank line after metadata)
```suggestion
    :keywords: tile source
    
'''
```